docs/2023/Sunbird UCI/Telemetry Dashboard/updates/2023-08-11.md

```diff
@@ -4,13 +4,17 @@ author: Pratiksha Sankhe
 ---
 
 ## Milestones
-- [ ] Give the description about Milestone 1
-- [ ] Give the description about Milestone 2
-- [ ] Give the description about Milestone 3
-- [ ] Give the description about Milestone 4
 
-## Screenshots / Videos 
+<ul>
+  <li>UI & APIS</li>
+  <input type="checkbox" disabled="" checked="true"/> Integrated the apis with the UI.<br/>
```

Comment:
There's no need of using tags
```suggestion
  <input type="checkbox" disabled="" checked="true"/> Integrated the apis with the UI.<br/>
  - [x] Integrated the apis with the UI
```

Reply:
I have used tags in all updates, so just to maintain the uniformity continuing it.